client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/index.js

```diff
@@ -4,91 +4,125 @@
  * External dependencies
  */
 import React from 'react';
-import { expect } from 'chai';
-import { configure, shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import { mount } from 'enzyme';
+import configureMockStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
 
 /**
  * Internal dependencies
  */
-import { CarrierAccounts } from '../index.js';
-import CarrierAccountListItem from '../list-item';
-import Card from 'components/card';
-import ExtendedHeader from 'woocommerce/components/extended-header';
+import CarrierAccounts from '..';
 
-configure( { adapter: new Adapter() } );
```

Comment:
No need to do this, the `tests/client/setup-test-framework.js` already takes care of it